docling_serve/datamodel/convert.py

```diff
@@ -296,6 +296,14 @@ class ConvertDocumentsOptions(BaseModel):
         ),
     ] = 2.0
 
+    md_page_break_placeholder: Annotated[
+        Optional[str],
+        Field(
+            description="Add this placeholder betweek pages in the markdown output.",
+            examples=["<!-- page-break -->", ""],
```

Comment:
I would remove the `""` from the examples and add `None` instead.

Reason: `""` is implemented in https://github.com/docling-project/docling-core/pull/298 to behave consistently with other strings, which will typically lead to surrounding newlines, which some users may not expect.